src/Settings.php

```diff
@@ -126,6 +132,14 @@ public static function parseArguments(array $arguments)
     {
         $arguments = new ArrayIterator(array_slice($arguments, 1));
         $settings = new self;
+        
+        $git = "git show HEAD \
+           --name-only \
+           --diff-filter=ACM \
+           -m \
+           --first-parent \
+           --format=format: \
+           -- "'*' . array('.php', '.phtml', '.php3', '.php4', '.php5')" | egrep -v '^$' || :";
```

Comment:
PHP Parallel Lint is supported on Windows too and this does not work on this OS. So I think, this must be reimplemented in PHP.